end-to-end/proposals_config.json

```diff
@@ -1,14 +1,14 @@
 {
-    "balancerLBP": {
+    "v2Phase1": {
         "deploy" : true,
         "exec" : false,
         "proposerAddress" : "0xe0ac4559739bD36f0913FB0A3f5bFC19BCBaCD52",
         "voterAddress" : "0xB8f482539F2d3Ae2C9ea6076894df36D1f632775"
     },
-    "upgrade" : {
+    "fip_10b": {
```

Comment:
fip 10b is the curve deposit which is currently separate from the main phase 1 deploy